news/20210323151402.misc

```diff
@@ -1 +1,2 @@
 Update form factor ARDUINO to ARDUINO_UNO in the custom targets travis test as ARDUINO has been deprecated.
+Add a CMake variable for the output image extension
```

Comment:
Can we add another, separate news fragment for this change? That way `towncrier` will format the `CHANGELOG.md` correctly when we make a release. We also want a `.bugfix` extension instead of `.misc` for this change, so the release pipeline bumps the version correctly.

Reply:
Thanks, done